charts/code-of-conduct.md

```diff
@@ -0,0 +1,133 @@
+
+# ODA Component Accelerator Code of Conduct
```

Comment:
"ODA Component Accelerator" -> "ODA Canvas"?

Reply:
Should be **ODA Accelerator**

Reply:
Changed. Thanks.